app/poros/quote.rb

```diff
@@ -0,0 +1,15 @@
+class Quote
+  attr_reader :author,
+              :content,
+              :categories
+
+  def initialize(quote_data)
+    @author = quote_data[:author]
+    @content = quote_data[:content]
+    if quote_data[:tags].nil?
+      return "N/A"
+    else
+      @categories = quote_data[:tags].join(", ")
+    end
+  end
```

Comment:
👍

Reply:
This is good. I think I might suggest that you pull lines 9-13 into an instance method. You could even call the instance method `categories`. I'd probably do that because we try to avoid putting logic in your initialize method.